src/hooks/use-webview-shared-props.tsx

```diff
@@ -47,10 +53,14 @@ export const useWebViewSharedProps = (
 
   useFocusEffect(() => {
     return () => {
-      const watchID = VariablesString.get(GEO_WATCH_ID_KEY);
-      if (watchID) {
-        Geolocation.clearWatch(Number(watchID));
-      }
+      VariablesString.getAll().forEach(({id, value}) => {
```

Comment:
Put this logic into VariablesString class.

Also this:
```
VariablesString.set(
        `${instanceId}:${GEO_WATCH_ID_KEY}:${watchID}`,
        String(geolocationWatchID),
      );
```
Should be in dedicated method. So in summary you will have: `VariablesString.[set/get]GeoID` and `VariablesString.removeAllGeoID`.